### DISCUSSION TITLE
Description, section variables

### DISCUSSION THREAD

**Discussion by Dieter-GitHub** [SukramJ/aiohomematic#980] [General]
Hi Daniel, Hi all,

in the section variables you write as order "Don't tag character strings with "hahm" and some lines later you describe that it is much better to mark with "hahm". This confuses me, and probably others, a little bit. Should you mean "..if not tagged.." instead of "Don't tag ..."?
Best regards

This is how it looks in the description:

This is how entities are created from system variables:

    all character strings are created as sensor entity. Don't tag character strings with hahm.
    DEFAULT: system variables that do not have the marker hahm in description:
        value list, number --> sensor entity
        alert, logic value --> binary_sensor entity
    EXTENDED: system variables that do have the marker hahm in description:
        value lists --> select entity
        number --> number entity
        alarm, logic value —> switch entity
        character string —> text entity (>= HA 2022.12)

Using select, number, switch and text results in the following advantages:

    System variables can be changed directly in the UI without additional logic.
    The general services for select, number, switch and text can be used.
    The service homematicip_local.set_variable_value can, but no longer has to, be used to write system variables.
    Use of device based automations (actions) is possible.

**Comment by SukramJ** [comment]
Thanks, i updated the readme.
